docs/application/dotnet/guides/nui/nui-components/Tab.md

```diff
@@ -7,46 +7,48 @@ A tab can contain one or more TabItem with text, usually used as a menu label. A
 ![Tab](./media/tab.png) ![Tab](./media/tab2.png)
 
 ## Create with property
+
 To create a tab using property, follow these steps:
+
 1. Create Tab using the default constructor:
 
-```
-utilityBasicTab = new Tab();
-```
+    ```cs
+    utilityBasicTab = new Tab();
+    ```
 
 2. Set the tab property

```

Comment:
Please add a colon at the end of the sentence, as it is following by a code block.